apps/parsley/src/components/SidePanel/NavGroup/FilterNavGroup/ShowFiltersToggle/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Toggle, { Size as ToggleSize } from "@leafygreen-ui/toggle";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { useLogWindowAnalytics } from "analytics";
+import { useFilterParam } from "hooks/useFilterParam";
+
+const ShowFiltersToggle: React.FC = () => {
```

Comment:
nit.  what do you think about `AllFiltersToggle`? (If you make this change don't forget the file name!)